cmd/server.go

```diff
@@ -58,10 +60,30 @@ var (
 	serverCmd = &cobra.Command{
 		Use:   "server [join-token]",
 		Short: "Run server",
+		Example: `	Command to associate master nodes:
```

Comment:
I don't know if it was an intention, but there is tab character here and later in the example strings

Reply:
It is required so when the example is printed it aligns correctly 
```
Usage:
  k0s server [join-token] [flags]

Examples:
	Command to associate master nodes:
	CLI agument:
	$ k0s server [join-token]

	or CLI flag:
	$ k0s server --token-file [path_to_file]
	Note: Token can be passed either as a CLI argument or as a flag
```
vs without tab
```
Usage:
  k0s server [join-token] [flags]

Examples:
Command to associate master nodes:
	CLI agument:
	$ k0s server [join-token]

	or CLI flag:
	$ k0s server --token-file [path_to_file]
	Note: Token can be passed either as a CLI argument or as a flag
```